pages/about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>Jean Emmanuel Cadet's Capstone Project</title>
+		<link rel="stylesheet" href="../styles/global.css">
+		<link rel="stylesheet" href="../styles/navbar.css">
+		<link rel="stylesheet" href="../styles/hero.css">
+		<link rel="stylesheet" href="../styles/main-program.css">
+		<link rel="stylesheet" href="../styles/speakers.css">
+		<link rel="stylesheet" href="../styles/partners.css">
+		<link rel="stylesheet" href="../styles/about.css">
+		<link rel="stylesheet" href="../styles/footer.css">
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+	</head>
+
+	<body>
+		<header>
```

Comment:
- [ ] Please also consider fixing the indentation for both your `index.html` and `about.html` pages. According to the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):

![image](https://github.com/jecode93/first-module-capstone-project/assets/77069011/31cad78a-e931-419c-87b0-ef323f043cf6)

Please consider fixing this! 🤓 🔧